kernel/src/driver/mod.rs

```diff
@@ -13,8 +13,11 @@ pub fn init() {
     // FIXME: Currently, we have to do this manually to ensure the crates containing the input
     // devices are linked and their `#[init_component]` hooks can run to register the devices with
     // the input core. We should find a way to avoid this in the future.
-    #[expect(unused_imports)]
-    use aster_keyboard::*;
+    #[cfg(all(target_arch = "x86_64", target_os = "none"))]
+    {
+        #[expect(unused_imports)]
+        use aster_i8042::*;
+    }
```

Comment:
What's wrong with:
```rust
    #[cfg(target_arch = "x86_64")]
    #[expect(unused_imports)]
    use aster_i8042::*;
```


Actually it seems that there is a more reasonable way: following #2529 to write [`extern crate xxx`](https://github.com/asterinas/asterinas/pull/2529/files#diff-794fada0ab41e6aff2843d93023ded466de0a93207551bbc74772a69c75a075d). Of course, feel free to view this as a separate issue and not address it in this PR.